test/data/apis.ts

```diff
@@ -102,7 +96,7 @@ export class MyService {
         @QueryParam("testOptional") test3?: string,
         @QueryParam("testEnum") test4?: TestEnum,
         @QueryParam("testNumericEnum") test5?: TestNumericEnum,
-        @QueryParam("testMixedEnum") test6?: TestMixedEnum
```

Comment:
why have you removed this test?

Reply:
Mixed enums will now throw an error as they are not supported by OpenAPI

Reply:
Should we test that?